react/react.py

```diff
@@ -96,7 +98,7 @@ def to_json(data):
         :return: Received data in json format
         :rtype: str
         """
-        if isinstance(data, str):
+        if isinstance(data, six.string_types):
```

Comment:
`json.dumps` accepts any serializable object. I think I would expect `to_json` to *work just as* `json.dumps`, that's why I plan to remove it later either way.

Checking for `six.string_types` instead of `six.text_type` will break the current behaviour for anyone passing bytes (in py3) to that function.